Automattic/jetpack
# Boost: Remove global state from Purchase Success component

## Proposed changes:

* Moves global state to parent component in favor of props;
* Add helper function to enable cloud css to avoid using global state in component.

### Other information:


## Jetpack product discussion



pc9hqz-286-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Make sure that opening the Purchase Success page enables Cloud CSS (you can check that in the network tab);
* Make sure that a new Image Size Analysis report is requested if image guide is active and you access the Purchase Success page.
